app/src/main/res/layout/topic_summary_view.xml

```diff
@@ -10,22 +11,19 @@
   </data>
 
   <androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
Changes to TopicName:

Add following attributes
`android:maxLines="2"`
`android:minLines="2"`
`android:ellipsize="end"`
`android:gravity="top"`

Change attribute to:
`android:layout_width="match_parent"`

Reply:
Updated changes.